hera_cal/redcal.py

```diff
@@ -218,6 +224,28 @@ def split_bls(bls):
         lengths = [np.mean([np.linalg.norm(antpos[bl[1]] - antpos[bl[0]]) for bl in gp]) for gp in reds]
         reds = [gp for gp, l in zip(reds, lengths) if ((min_bl_cut is None or l > min_bl_cut)
                                                        and (max_bl_cut is None or l < max_bl_cut))]
+    
```

Comment:
I think the block of code below is hard too follow. Would it be possible to add a comment paragraph paragraph or two describing how it checks degrees of freedom and makes sure that the degeneracies do not end up exceeding  max_dims and then in each line add a short comment tying to the english narrative that is layed out int he paragraph?